aws_compliance.py

```diff
@@ -373,6 +373,49 @@ def unused_credentials(credreport):
                 pass
     return {'Result': result, 'failReason': failReason, 'Offenders': offenders, 'ScoredControl': scored, 'Description': description, 'ControlId': control}
 
+def old_api_keys(credreport):
+    """Summary
+
+    Args:
+        credreport (TYPE): Description
+
+    Returns:
+        TYPE: Description
+    """
+    result = True
+    failReason = ""
+    offenders = []
+    control = "old_api_keys"
+    description = "Ensure API keys older than 90 days are rotated."
+    scored = False
+    # Get current time
+    now = time.strftime('%Y-%m-%dT%H:%M:%S+00:00', time.gmtime(time.time()))
+    frm = "%Y-%m-%dT%H:%M:%S+00:00"
+
+    # Look for unused credentails
+    for i in range(len(credreport)):
+        if credreport[i]['access_key_1_active'] == "true":
+            try:
+                delta = datetime.strptime(now, frm) - datetime.strptime(credreport[i]['access_key_1_last_rotated'], frm)
+                # Verify password have been used in the last 90 days
+                if delta.days > 90:
+                    result = False
+                    failReason = "API key older than 90 days detected."
+                    offenders.append(str(credreport[i]['arn']) + ":key1")
+            except:
+                pass
+        if credreport[i]['access_key_2_active'] == "true":
+            try:
+                delta = datetime.strptime(now, frm) - datetime.strptime(credreport[i]['access_key_2_last_rotated'], frm)
+                # Verify password have been used in the last 90 days
+                if delta.days > 90:
+                    result = False
+                    failReason = "API key older than 90 days detected."
+                    offenders.append(str(credreport[i]['arn']) + ":key2")
```

Comment:
```suggestion
                    offenders.append(f'{str(credreport[i]['arn'])}:key2')
```

nit but maybe better to use string interpolation?